src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
 - `js-beautify` for pretty generated HTML.
 
 - `json5` for more human readable representation of arbitrary values.
+
+---
+
+## Microdata and semantic tag usage (WIP)
+
+### Schema nodes to add microdata to
+* Article
+    * `div` `itemtype='schema:Article'`
+    * itemprops:
+        * authors* (author?): `ol`
+        * title*: `h1`? (+ `title`)
+        * environment: `data`?
+* Audio
+    * `audio`
+    * itemprops:
+        * caption*: `figcaption`
+        * transcript
```

Comment:
Right now both `caption` and `transcript` are simple strings. Something to evaluate would be to:
- either make `transcript` be a nested `CreativeWork` or an `Article`
- change `caption` conform to a `uri string` and reference a `.vtt` file. This way we can use the HTML [`<track> tag`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track). Reason I say this is that I don't think I'd want to avoid coupling the `audio` tag to always be inside a `figure` element

Reply:
I think I may have confused the `caption` and `name` properties for AudioObject.

- Re: `figcaption` usage, I was looking at [this example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio). Maybe instead of `caption` it should be `name` that gets encoded as a `figcaption`. Or we could use a `p`, or preserve the name in `data`.
- Re: `track` usage, I don't believe we can add captions/transcripts the same way in `audio` as we can in `video`. This may be a bit in the weeds, but a workaround is to [use `video` tags instead of `audio](https://www.iandevlin.com/blog/2015/12/html5/webvtt-and-audio/), or display the captions/transcripts separately.
- Both those changes make sense: `caption` conforming to a `uri string`/referencing a `.vtt` file, and changing transcript to a nested `CreativeWork`/`Article`.